src/main/java/core/basesyntax/Application.java

```diff
@@ -2,7 +2,17 @@
 
 public class Application {
     public static void main(String[] args) {
-        // test your code here ...
+        Engine engine = new Engine("JZ-1",600,550,9.5,7);
 
+        Engine engineClone = engine.clone();
```

Comment:
we don't need to clone the engine in `main` method - we should only clone the car

and then change one of the properties of the engine of the original car